server/app/services/export/AnswerDataConverter.java

```diff
@@ -0,0 +1,121 @@
+package services.export;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import services.CfJsonDocumentContext;
+import services.Path;
+import services.applicant.AnswerData;
+import services.applicant.question.Question;
+import services.question.types.QuestionType;
+
+public final class AnswerDataConverter {
+  private static final Logger logger = LoggerFactory.getLogger(AnswerDataConverter.class);
+
+  private static final ImmutableSet<QuestionType> USE_APPLICATION_PATH_TYPES =
+      ImmutableSet.of(
+          QuestionType.NAME,
+          QuestionType.ID,
+          QuestionType.TEXT,
+          QuestionType.EMAIL,
+          QuestionType.ADDRESS,
+          QuestionType.DROPDOWN,
+          QuestionType.RADIO_BUTTON,
+          QuestionType.FILEUPLOAD);
```

Comment:
`USE_APPLICATION_PATH_TYPES` is based on the paths where we used to call `asApplicationPath()` on the switch statement:

https://github.com/civiform/civiform/blob/358ad3dbb6e67aacb91afb3ede60e32ac281ba26/server/app/services/export/JsonExporter.java#L121-L219